### PR TITLE
Centerized styling

### PR DESCRIPTION
**Pull request by khanh-devos**
Add a centerized styling to the new reservation form.
Set latest bike as default at the new reservation form.

### CODE REVIEW

File: src/Components/reservations/NewReservation.jsx [hunk 1]
```diff
@@ -58,23 +60,17 @@ export default function NewReservation() {
   return (
     <div className="reservation-page">
 
-      <img
-        className="reservation-page-background"
-        alt="reservarion-background"
-        src={bike?.image || 'https://cdn.shopify.com/s/files/1/1439/6088/files/thin.jpg?width=100;height:100'}
-      />
+      <MirrorCover bike={bike} />
```
Comment (souad988) [inline src/Components/reservations/NewReservation.jsx]: @khanh-devos  good job so far, I want just to suggest using a white background here to stick as possible to the design also please use the same colors used there

File: src/Components/reservations/NewReservation.jsx [hunk 2]
```diff
@@ -58,23 +60,17 @@ export default function NewReservation() {
...
 
-      <div className="reservation-page-bg-cover" />
-
-      <h2><strong>NEW RESERVATION</strong></h2>
+      <h2 className="auth-title reservation-title">BOOK A TEST-RIDE</h2>
```
Comment (souad988) [inline src/Components/reservations/NewReservation.jsx:65]: also can you add the view or just in this same component let user select location first then move them to the form to select duration as shown in the design 
Capture d’écran du 2023-11-13 14-31-58

File: src/Components/reservations/cover.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <>
+      <img
+        className="reservation-page-background"
+        alt="reservarion-background"
+        src={bike?.image || 'https://cdn.shopify.com/s/files/1/1439/6088/files/thin.jpg?width=100;height:100'}
+      />
+
+      <div className="reservation-page-bg-cover" />
+    </>
```
Comment (souad988) [inline src/Components/reservations/cover.jsx]: can you check here this image doesn't show up in my end 
Capture d’écran du 2023-11-13 14-28-24

**Review by souad988 — CHANGES_REQUESTED** [review]
@khanh-devos  good improvements so far 👍🏼 , please check the comments below before we can approve these changes.

**Comment by rodrigopenayo98** [comment]
Everything ready @khanh-devos ... nothing more to add, ready to do the merge!

**Pull request merged**